### DISCUSSION TITLE
Combine both OpenID and regular login?

### DISCUSSION THREAD

**Discussion by nanawel** [FreshRSS/FreshRSS#6986] [Q&A] [answered]
Hi,
I've just set up Authentik successfully but I was wondering if there was a way to allow OpenID _and_ the regular login method to be usable at the same time?

Typical use cases are 
- to be able to log into FreshRSS if Authentik is down
- to use a local user who does not exist in Authentik

Most of the apps have this choice on the login page (login + password and a "Login with SSO/OpenID/etc." button).

**Comment by Alkarex** [comment, marked as answer]
This is not currently possible out of the box. 
I believe it would require making a second entry point, one for each login method. At the moment, there is a single one `/i/` that supports only a single login method at a time. 
For now, the easiest is likely to run two FreshRSS instances in parallel.
